sw/device/silicon_creator/rom_ext/sival/binaries/BUILD

```diff
@@ -53,9 +53,25 @@ filegroup(
     srcs = glob(["*.signed.bin"]),
 )
 
+filegroup(
+  name = "signed_spx_binaries",
+  srcs = ["//sw/device/silicon_creator/rom_ext:rom_ext_dice_x509_slot_a"],
+  output_group = "fpga_cw340_signed_bin",
+  testonly = True,
+)
+
 signature_test(
     name = "sival_signature_test",
+    spx_domain = "", # "Pure", "PrehashedSha256", "PrehashedSha256Reversed"
```

Comment:
This is the default value - I'd say we omit it, since we don't want to perform SPX verification of the SiVAL images as they aren't signed with SPX signatures.